documentation/projects.md

```diff
@@ -49,29 +25,41 @@ To get started with a Bolt project:
    name: myproject
    ```
 3. Develop your Bolt plans and tasks in `plans` and `tasks` directories in the
-   root of the project directory, next to `bolt-project.yaml`.
+   root of the project directory, next to `bolt-project.yaml`. Bolt loads tasks
+   and plans from the `tasks` and `plans` directories and namespaces them to the
+   project name.
 
-Bolt treats a directory as a Bolt project as long as a `bolt-project.yaml` file
-exists at the root of the directory and contains a `name` key. Bolt loads tasks
-and plans from the `tasks` and `plans` directories and namespaces them to the
-project name.
+Here is an example of a typical project with a task, a plan, and an inventory
+file:
+
+```console
+.
+├── bolt-project.yaml
+├── inventory.yaml
+├── plans
+│   └── myplan.yaml
+└── tasks
+    ├── mytask.json
+    └── mytask.py
+```
 
 ## Configuring a project
 
-The `bolt-project.yaml` file holds options to configure your Bolt project, as
-well as options to control how Bolt behaves when you run a project.
+Besides the `name` key, the `bolt-project.yaml` file holds options to configure
+your Bolt project, as well as options to control how Bolt behaves when you run a
+project.
 
-For example, if you wanted Bolt to load a custom module path, you could use the
-following in your `bolt-project.yaml`:
+For example, if you want to add a little flare to your Bolt output, use the
```

Comment:
```suggestion
For example, if you want to add a little flair to your Bolt output, use the
```

I don't know if we want people to light their Bolt output on fire 😉

Reply:
Rofl. Good catch